### PR TITLE
Chef 15 Fixes

### PR DESCRIPTION
**Pull request by zimmerry**

### CODE REVIEW

**Comment by zimmerry** [comment]
`execute[enable-dummy-nics]` in the test cookbook causes idempotency issues in the haproxy and mysql2 suites, not sure how to fix that.

**Comment by ramereth** [comment]
> `execute[enable-dummy-nics]` in the test cookbook causes idempotency issues in the haproxy and mysql2 suites, not sure how to fix that.

You might see the suggestion I made in osuosl-cookbooks/osl-dns#39

File: .rubocop.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ AllCops:
   TargetChefVersion: 15.latest
   Exclude:
     - 'cookbooks/**/*'
+    - 'test/integration/**/*'
```
Comment (ramereth) [inline .rubocop.yml:5]: Why was this added?

Comment (zimmerry) [reply .rubocop.yml:5]: There are tabs in the integration tests for haproxy_osuosl, haproxy_phpbb, and mysql2. When I replace the tabs with spaces, the InSpec tests fail.

Comment (ramereth) [reply .rubocop.yml:5]: So instead of excluding the whole files I recommend you only disable the check for the sections of code it complains about by doing it this way:
```ruby
# rubocop:disable Layout/IndentationStyle
bad code
# rubocop:enable Layout/IndentationStyle
```
You'll have to do this in multiple places in the same file.

**Comment by zimmerry** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v0.4.0.  Have a nice day!
